### ISSUE TITLE
Consider making fixture runner public/official

### ISSUE THREAD

**Issue by Andarist** [babel/babel#6295]
**feature request**

### Context
At the moment those modules:
- `babel-helper-fixtures`
- `babel-helper-plugin-test-runner`
- `babel-helper-transform-fixture-test-runner`

Have this note:
> **NOTE:** This is an internal Babel module and may not work outside. Use at your own risk.

I think the community of babel plugin writers could benefit from having those packages as officially supported. Probably already some depend on it anyway - but claiming it as stable would be nice.

**Comment by xtuc** [comment]
Yes I agree. I did actually try this in the past and IRC the issue was that is depends on the Lerna packages structure. It's not easy to have something generic enough.

I could image a very generic test runner and a way to launch it with Lerna, does it make sense to you?

**Comment by hzoo** [comment]
there's also https://github.com/babel-utils/babel-plugin-tester, and jest, etc. We should figure out internally too
